### PR TITLE
Update jquery.ime.test.fixtures.js

### PR DESCRIPTION
**Pull request by tuxnani**
Added test cases for https://github.com/tuxnani/jquery.ime/commit/453f6326bb16e60990e1516aa8c29c4bad225594
Added required test cases for adding jna gunintam to Telugu-transliteration

### CODE REVIEW

File: test/jquery.ime.test.fixtures.js [hunk 1]
```diff
@@ -2986,5 +2986,13 @@ var testFixtures = [
 			{ input: 'Ẑh', output: 'Zh', description: 'Venetian Zh' }
 		],
 		inputmethod: 'vec-GVU'
+	},
+	{
+		description: 'Telugu Transliteration test',
+		tests: [
+			{ input: 'j~nu', output: 'జ్ఞు', description: Telugu j~nu -> జ్ఞు' },
+			{ input: 'j~nE', output: 'జ్ఞే', description: Telugu j~nE -> జ్ఞే' },
```
Comment (santhoshtr) [inline test/jquery.ime.test.fixtures.js]: Please remove trailing comma here. Also missing quote for description
Also please test once to see all tests pass.

**Comment by tuxnani** [comment]
Removed trailing comma and added lines at relevant place.

File: test/jquery.ime.test.fixtures.js [hunk 2]
```diff
@@ -1809,7 +1809,9 @@ var testFixtures = [
 		description: 'Telugu Transliteration test',
 		tests: [
 			{ input: 'c', output: 'చ్', description: 'Telugu c' },
-			{ input: 'ch', output: 'చ్', description: 'Telugu ch' }
+			{ input: 'ch', output: 'చ్', description: 'Telugu ch' },
+			{ input: 'j~nu', output: 'జ్ఞు', description: Telugu j~nu -> జ్ఞు' },
```
Comment (kartikm) [inline test/jquery.ime.test.fixtures.js]: Missing single quote.

**Comment by amire80** [comment]
Replaced by #508.

**Pull request closed without merging**